usim/py/exceptions.py

```diff
@@ -0,0 +1,40 @@
+"""
+This module defines the exceptions exclusive to the SimPy compatibility layer.
+These are used as shutdown signals between the environment and processes.
+Note that with the exception of :py:exc:`~usim.py.exceptions.Interrupt`,
+all signals should be considered internal and not meant to be handled manually.
+"""
+
+
+class NotEmulatedError(NotImplementedError):
```

Comment:
This Error we should consider for renaming. As mentioned in several other comments, the compatibility layer does not do a proper emulation but a simulation. However, talking about simulation error inside a simulation might be confusing for a user. So we should think about a proper name.